x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/details_page/components/package_policies/package_policies_table.tsx

```diff
@@ -189,6 +189,7 @@ export const PackagePoliciesTable: React.FunctionComponent<Props> = ({
                   </EuiFlexItem>
                   <EuiFlexItem grow={false}>
                     <EuiButton
+                      data-test-subj="PackagePoliciesTableUpgradeButton"
```

Comment:
nit: should we agree on a convention of naming `data-test-subj`? I mean to use camelCase everywhere.

Reply:
I think so, I went with camel case but I noticed that there is no single convention. We should discuss it with the team maybe.